server/common/auth.go

```diff
@@ -17,6 +18,8 @@ type UserClaims struct {
 	jwt.RegisteredClaims
 }
 
+var invalidatedTokenCache = cache.NewMemCache[bool]()
```

Comment:
Preserving all valid tokens could be a preferable choice as it prevents invalid tokens from being revalidated upon restart. What are your thoughts?

Reply:
Yeah that seemed to be a better choice, I will update my code.